engines/ic3ia.cpp

```diff
@@ -202,10 +202,14 @@ void IC3IA::initialize()
   // NOTE: abstract is called automatically in IC3Base initialize
   UnorderedTermSet preds;
   get_predicates(solver_, ts_->init(), preds, false);
+  size_t num_init_preds = preds.size();
   get_predicates(solver_, bad_, preds, false);
+  size_t num_prop_preds = preds.size() - num_init_preds;
   for (const auto &p : preds) {
     add_predicate(p);
   }
+  logger.log(1, "Number predicates found in init: {}", num_init_preds);
+  logger.log(1, "Number predicates found in prop: {}", num_prop_preds);
   // more predicates will be added during refinement
```

Comment:
perhaps, let's also print the total number of predicates we add initially.

Reply:
Cool, added with 4c4251db7f7e4effea18f6b0bff5287bc4765316